.changes/1028.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "description": "link to libgcc for armv5te-unknown-linux-musleabi.",
+        "type": "fixed"
+    },
+    {
+        "description": "add C++ support for FreeBSD targets.",
+        "type": "added"
+    },
+    {
+        "description": "test dynamic library support for Android targets in CI.",
+        "type": "internal"
+    },
+    {
+        "description": "test partial C++ support for mips64el-unknown-linux-muslabi64 in CI.",
+        "type": "internal"
+    },
+    {
+        "description": "convert mips64el-unknown-linux-muslabi64 to a hard-float toolchain to match the rust target.",
+        "type": "changed",
+        "breaking": true
+    },
+    {
+        "description": "convert mips64el-unknown-linux-muslabi64 to use the mips64r2 architecture, identical to the rust target.",
+        "type": "changed",
+        "breaking": true
+    },
+    {
+        "description": "document whether MIPS musl targets are hard or soft float.",
+        "type": "internal"
+    },
```

Comment:
This isn't really documented, only commented in the respective dockerfiles. I think this entry is not needed